src/bflib_video.c

```diff
@@ -968,6 +968,18 @@ TbPixel LbPaletteFindColour(const unsigned char *pal, unsigned char r, unsigned
     }
     return *o;
 }
+
```

Comment:
Is this a good place to add this function? I will make use of it throughout the KFX codebase as I continue to fix the scaling issues.